webapp/sdf/src/components/item-component.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <h2>{{ item.title }} ({{ item.votes }})</h2>
+    <button v-on:click="updateItem(item, 1)">Upvote</button>
+    <button @click="updateItem(item, -1)">Downvote</button>
+    <button v-on:click="removeItem(item)">Remove</button>
+  </div>
+</template>
+
+<script>
+import Item from "../classes/item";
+export default {
+  name: "ItemComponent",
+  props: {
+    item: Item,
+  },
+  methods: {
+    removeItem: function (_item) {
```

Comment:
```suggestion
    removeItem(_item) {
```

Use shorthand: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions#Description